.github/workflows/update-formula.yml

```
@@ -0,0 +1,43 @@
+name: Update Homebrew Formula
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  update-formula:
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-24.04
```

We use fixed version by convention.